Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:10
+
+RUN mkdir -p /app/envolution
```

Comment:
It's not necessary to create it, with WORKDIR this folder will be automatically setted.